README.md

```diff
@@ -1,18 +1,23 @@
 # UpCloud Java API client library
 
+> **DEPRECATION NOTICE**
+>
+> This repo is deprecated and won't receive more fix, feature or version updates. We suggest using our other SDK:s like [UpCloud Go API](https://github.com/UpCloudLtd/upcloud-go-api) or [UpCloud Python API](https://github.com/UpCloudLtd/upcloud-python-api). Or just use [UpCloud API](https://developers.upcloud.com/1.3/1-introduction/) directly.
```

Comment:
Maybe 'this SDK' or 'this API' has been deprecated? The repo itself isn't deprecated, I guess?

Also, a few language suggestions;

> .. won't receive more fix*es*, feature*s* or ..
> like UpCloud Go API, UpCloud Python API or just call UpCloud API directly

Although, the last change sort of raises the question/opens a new can of works (that probably is out of scope for this discussion) if we should even be calling these APIs, it's a bit confusing as it is.